src/BigBlueButton.php

```diff
@@ -443,14 +489,59 @@ public function setJSessionId($jSessionId)
         $this->jSessionId = $jSessionId;
     }
 
+    private function isJson($string)
+    {
+        return is_string($string) && is_array(json_decode($string, true)) && (json_last_error() == JSON_ERROR_NONE) ? true : false;
```

Comment:
Why is `is_array` required? `"true"` is also valid JSON.

Is `? true : false` really required?

Reply:
bigbluebutton return as json so check is valid json or not so i added that function for testing